src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -136,7 +136,23 @@ const createFormValidator = celebrate({
       })
       .required()
       // Allow other form schema keys to be passed for form creation.
-      .unknown(true),
+      .unknown(true)
+      .custom((value, helpers) => {
+        // If there are invalid utf-8 encoded unicode-escaped characters,
+        // node 14 treats the sequence of characters as a string e.g. \udbbb is treated as a 6-character string instead of an escaped unicode sequence
```

Comment:
We are now on node 18, does this still hold?

Reply:
Yup it still holds:
```javascript
➜  ~ node --version
v18.12.0

➜  ~ node
Welcome to Node.js v18.12.0.
Type ".help" for more information.
> JSON.stringify('\u4E24') // valid unicode sequence
'"两"'
> JSON.stringify('\udbbb') // invalid unicode sequence
'"\\udbbb"'
```

Reply:
Maybe we can replace `node 14` all the comments by simply `nodejs`.

Reply:
Thanks for helping to check! @timotheeg 

Updated the comments here [8e6b94b](https://github.com/opengovsg/FormSG/pull/5436/commits/8e6b94b9bee14935c52c05d8974ec8cac48a1f80)